.github/ISSUE_TEMPLATE/story.yaml

```diff
@@ -0,0 +1,45 @@
+name: Story (Internal Use Only 🔒)
+description: Define a user story.
+title: "[Story]: <DESCRIPTIVE TITLE HERE>"
+labels: ["type/user-story"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## ⚠️ Warning
+        Issues that do not follow the below template will be marked `status/back-to-author` and assigned back to the creator.
+        
+        ---
+        
+        # 📚️ Story Details
+  - type: textarea
+    id: user-stories
+    attributes:
+      label: The user story
+      description: Explain the situation in [user story format](https://www.atlassian.com/agile/project-management/user-stories).
+      value: |
+        As a <PERSONA>, I want <ENHANCEMENT> so that <REASONING>.
+
+        I know this is done when:
+          - <ACCEPTANCE CRITERIA #1>
+          - ...
+      render: bash
+    validations:
+      required: true
+  - type: input
+    id: context
+    attributes:
+      label: Epic link
+      description: Provide the hyperlink of the over-arching epic.
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        
+        ## What's next?
+        A member of the AWS SDK Example Code will review your issue and reply within 10 business days. In the meantime, we discourage editing the issue but welcome structural edits and additional details! 🙂
```

Comment:
Doesn't really make sense for internal use issues.

Reply:
Removed.